Linux/sgx/libsgx_tsgxssl/tdefines.h

```diff
@@ -31,30 +31,15 @@
  
 #ifndef __TDEFINES_H__
 #define __TDEFINES_H__
-
+#include "time.h"
+
 #define TRUE	1
 #define FALSE	0
 
 typedef unsigned int socklen_t;
 
 typedef long int time_t;

```

Comment:
Can we remove this definition as well?

Reply:
Yes. It seems definitions of  `time_t`, `pthread_rwlock_t`, `pthread_rwlockattr_t`, `pthread_once_t`, `pthread_key_t`, `pthread_t` can be removed, when including the time.h and pthread.h from SGX SDK.